opennms-doc/guide-development/src/asciidoc/text/rest/alarms.adoc

```diff
@@ -46,3 +46,18 @@ In OpenNMS 1.11.0, some additional features are supported in the alarm ack API:
 
 Additionally, when acknowledging alarms (ack=true) you can now specify an `ackUser` parameter.
 You will only be allowed to `ack` as a different user IF you are PUTting as an authenticated user who is in the _admin_ role.
+
+===== v2 API
+
+Since Horizon 20.1.0 an alarms endpoint has been available in the v2 API which uses link:https://github.com/jirutka/rsql-parser#rsql--fiql-parser[FIQL] for querying.
```

Comment:
```suggestion
Since Horizon 20.1.0, an alarms endpoint has been available in the v2 API that uses link:https://github.com/jirutka/rsql-parser#rsql--fiql-parser[FIQL] for querying.
```